tests/blocks/test_core.py

```diff
@@ -2477,3 +2477,30 @@ class Baz_Alias(Block):
 
         # new local schema ID should now be saved to Prefect
         assert updated_schema_id == new_schema_id
+
+
+class TestDeleteBlock:
```

Comment:
Could you also add a test covering `.delete()` when used with the base `Block` class? (e.g. `Block.delete("newblock/my-block")`)

Reply:
Great catch @desertaxle ! Added the test now! Thanks a ton for the input! 🙌